lib/active_record/connection_adapters/trilogy_adapter.rb

```diff
@@ -155,6 +155,12 @@ def initialize_type_map(m)
       end
 
       def initialize(connection, logger, connection_options, config)
+        config = config.dup
+
+        # Trilogy ignores `socket` if `host is set. We want the opposite to allow
```

Comment:
Missing a closing back-tick after "host".  Should be:

```
# Trilogy ignores `socket` if `host is set. We want the opposite to allow
```

Reply:
Anyway, I shouldn't worry -- the same typo is also in the original!

Reply:
I updated commit to fix the typo. Thanks for pointing this out!